transitions/style.scss

```diff
@@ -0,0 +1,112 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-size: 62.5%;
+}
+
+body {
+    background-color: silver;
+}
+
+.title {
+    position: absolute;
+    display: block;
+    width: 100%;
+    text-align: center;
+
+    p {
+        font-size: 5rem;
+        text-transform: uppercase;
+        color: #fdcb9e;
+        text-shadow: 0.5rem 0.5rem 0.5rem #3f3f44;
+    }
+
+    z-index: 2;
+}
+
+.images {
+    display: flex;
+    height: 58rem;
+    width: 100%;
+    flex-direction: row;
+
+}
+
+.image {
+    display: flex;
+    flex: 1;
+    justify-content: center;
+    align-items: center;
+    transition: font-size 0.7s ease-in-out,
+        flex 0.7s cubic-bezier(.15, -0.64, .36, .94), filter 0.2s;
+    filter: grayscale(87%) sepia(28%) saturate(0%) contrast(134%);
+
+    p {
+        font-size: 4rem;
+        font-weight: 800;
+        color: #cceabb;
+        text-shadow: 0.5rem 0.5rem 0.5rem #3f3f44;
+    }
+
+}
+
+.image.open {
+    flex: 3;
+    filter: none;
+
+    p {
+        font-size: 5rem;
+        text-transform: uppercase;
```

Comment:
I know this is for fun, but usually, we should not animate `text-transform`. The animation doesn't get that smooth to the eyes because of the instant typography changing. :)